src/prices/PricesUpdater.ts

```
@@ -155,13 +156,33 @@ async function addStakedCelo(prices: PriceByAddress) {
   return returnedPrices
 }
 
+async function wrapCoingeckoCall(
```

I'd probly move this to the coingecko module

---

also, do you ever want to call `fetchPrices` without wrapping it? not sure it needs its own layer of abstraction

---

Sure, I'll move it to the coingecko file.
But yeah, I think the coingecko service should throw an error when it can't fetch prices, and leave the responsibility to handle that error to the consumer. For now, we want to ignore the error, but that might change, so ignoring the error in the service as default didn't sound like the best abstraction.